Sources/Base/NVActivityIndicatorView.swift

```diff
@@ -66,7 +66,7 @@ import UIKit
  - AudioEqualizer:          AudioEqualizer animation.
  - CircleStrokeSpin:        CircleStrokeSpin animation.
  */
-public enum NVActivityIndicatorType: CaseIterable {
+@objc public enum NVActivityIndicatorType: Int, CaseIterable {
```

Comment:
This will, unfortunately, break this https://github.com/ninjaprox/NVActivityIndicatorView/blob/52262639dc752b0e73b11c53e06edc43ccb27865/Sources/Base/NVActivityIndicatorView.swift#L535